packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -480,6 +481,947 @@ describe('postgresql - missing database', () => {
+        console.error(e)
+      })
+      // Back to original env vars
+      process.env = { ...originalEnv }
+      // Update env var because it's the one that is used in the schemas tested
+      process.env.TEST_POSTGRES_URI_MIGRATE = connectionString
+    })
+
+    afterEach(async () => {
+      // Back to original env vars
+      process.env = { ...originalEnv }
+      await tearDownPostgres(setupParams!).catch((e) => {
+        console.error(e)
+      })
+    })
+
+    test('basic introspection', async () => {
+      ctx.fixture(`introspection/postgresql/views-warning-${warningCode}`)
+      const introspect = new DbPull()
+      const result = introspect.parse(['--print'])
+      await expect(result).resolves.toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+        generator client {
+          provider        = "prisma-client-js"
+          previewFeatures = ["views"]
+        }
+
+        datasource db {
+          provider = "postgres"
+          url      = env("TEST_POSTGRES_URI_MIGRATE")
+        }
+
+        model User {
+          id         Int     @id @default(autoincrement())
+          first_name String  @db.VarChar(255)
+          last_name  String? @db.VarChar(255)
+        }
+
+        /// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+        view Schwuser {
+          id         Int?
+          first_name String? @db.VarChar(255)
+          last_name  String? @db.VarChar(255)
+
+          @@ignore
+        }
+
+
+        // introspectionSchemaVersion: NonPrisma,
+      `)
+      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+
+                                                                                                                // *** WARNING ***
+                                                                                                                // 
+                                                                                                                // The following views were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client.
+                                                                                                                // - View "Schwuser"
+                                                                                                                // 
+                                                                                    `)
+      expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
+  })
+
+  describe('postgresql views 25', () => {
```

Comment:
What happens here that is specific to code 25?

Reply:
Currently nothing, the original intent was to figure out what was happening with the cli warnings only emitting code 24 for introspection but given that it's intended for spam reasons (and in alignment with models), this could be removable now. Same goes for the introspecting 24 test

Reply:
I do not understand what "spam reasons" means. 24 and 25 have to do _something_ special that the others do not, or it does not make sense that they exist.

Reply:
Ah, that we don't emit too many warnings at once basically :)